bootstrap.go

```diff
@@ -384,6 +390,44 @@ func upgradeToVer10(s Session) {
 	doReentrantDDL(s, "ALTER TABLE mysql.stats_histograms DROP COLUMN use_count_to_estimate", ddl.ErrCantDropFieldOrKey)
 }
 
+func upgradeToVer11(s Session) {
+	s.Execute("BEGIN")
+	sql := "SELECT user, host, password FROM mysql.user WHERE password != ''"
+	rs, err := s.Execute(sql)
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+	r := rs[0]
+	sqls := make([]string, 1)
+	defer r.Close()
+	row, err := r.Next()
+	for err == nil && row != nil {
+		user := row.Data[0].GetString()
+		host := row.Data[1].GetString()
+		pass := row.Data[2].GetString()
+		newpass, err := util.OldPasswordUpgrade(pass)
+		if err != nil {
+			log.Fatal(err)
+			return
+		}
+		sql := fmt.Sprintf(`UPDATE mysql.user set password = "%s" where user="%s" and host="%s"`, newpass, user, host)
+		sqls = append(sqls, sql)
+		row, err = r.Next()
+	}
+
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+
+	for _, sql := range sqls {
+		mustExecute(s, sql)
+	}
+
```

Comment:
Add	`updateBootstrapVer(s)` here.
This upgrade is not reentrant, right? So it must be success/fail along with bootstrap version change.

Rest LGTM
PTAL @shenli